pkg/cmd/grafana-cli/services/api_client.go

```diff
@@ -118,32 +153,23 @@ func sendRequest(client http.Client, repoUrl string, subPaths ...string) ([]byte
 	req.Header.Set("grafana-arch", runtime.GOARCH)
 	req.Header.Set("User-Agent", "grafana "+grafanaVersion)
 
-	if err != nil {
-		return []byte{}, err
-	}
-
-	res, err := client.Do(req)
-	if err != nil {
-		return []byte{}, err
-	}
-	return handleResponse(res)
+	return req, err
```

Comment:
Do you mind returning the error at line 149 if it isn't nil? Doesn't feel safe to manipulate `req` in case an error was returned.